shared/src/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -224,6 +225,65 @@ Status copy() {
         }
     }
 
+    /**
+     * Document ended progress of a replicator.
+     */
+    public final static class DocumentReplicatedStatus {
```

Comment:
1. The correct name is DocumentReplication.
2. The correct API is here : https://github.com/couchbaselabs/couchbase-lite-apiv2/blob/master/spec/spec/Replicator/Replicator.swift#L67-L72.
3. Please extract the class to its own java file.

Reply:
After all, I think we don't need this class as there is another one below called `DocumentReplicatedUpdate`.